### PR TITLE
Adapt ncurses

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

**Comment by DimStar77** [comment]
(bcs#1074600) -> (bsc#1074600)

**Comment by DimStar77** [comment]
and imho the commits should be squashed - but I don't know if you guys have the same rules there :)

**Comment by jreidinger** [comment]
@DimStar77 not so strict, we have it separated, as it is later easier to do git revert. We do not want to revert changes and version increase, just code.

File: src/ncursesw.h [hunk 1]
```diff
@@ -405,6 +405,12 @@ inline int UNDEF( insertln )()	{ return insertln(); }
 #define insertln UNDEF(insertln)
 #endif
 
+#ifdef is_linetouched
```
Comment (jreidinger) [inline src/ncursesw.h]: Note, I took this code from upstream ncursesw. To be honest I do not know why we have partial copy of that file there we just have subset of this macro redefinition and modify some method signatures to act as shortcut, but overall I do not believe it is proper way how to do it.

**Comment by jreidinger** [comment]
superseeded by https://github.com/libyui/libyui-ncurses/pull/62

**Pull request closed without merging**